### PR TITLE
flannel tweaks for running with kube ansible

### PR DESCRIPTION
**Pull request by jasonbrooks**
* add cap for reading cert dir
* mount config/certs dirs
* copy and source config file from rpm

### CODE REVIEW

**Comment by giuseppe** [comment]
LGTM!

File: flannel/flanneld-run.sh [hunk 1]
```diff
@@ -4,10 +4,12 @@
 echo "[Service]" > /etc/systemd/system/docker.service.d/$NAME.conf
 echo "EnvironmentFile=-/run/$NAME/docker" >> /etc/systemd/system/docker.service.d/$NAME.conf
 
+source /etc/sysconfig/flanneld
+
 # Ensure this file doesn't already exist.
 rm -f run/flannel/subnet.env
 
-NOTIFY_SOCKET=/dev/null /usr/bin/flanneld &
+NOTIFY_SOCKET=/dev/null /usr/bin/flanneld -etcd-endpoints=${FLANNEL_ETCD_ENDPOINTS} -etcd-prefix=${FLANNEL_ETCD_PREFIX} -etcd-cafile=${FLANNEL_ETCD_CAFILE} -etcd-certfile=${FLANNEL_ETCD_CERTFILE} -etcd-keyfile=${FLANNEL_ETCD_KEYFILE} $FLANNEL_OPTIONS &
```
Comment (strigazi) [inline flannel/flanneld-run.sh]: We could leave those in FLANNEL_OPTIONS to be able to use etcd insecurely.

Comment (strigazi) [inline flannel/flanneld-run.sh]: I guess no

Comment (jasonbrooks) [inline flannel/flanneld-run.sh]: I just tested it w/o providing values for the ca, cert, keyfile, and was able to use etcd insecurely.

Comment (strigazi) [inline flannel/flanneld-run.sh]: Thanks

**Comment by giuseppe** [comment]
@strigazi fine to merge?

**Comment by strigazi** [comment]
Since it works without passing the certificates, it is fine for me.

Thanks

**Comment by giuseppe** [comment]
@rh-atomic-bot r+ 109f393

**Pull request closed without merging**